### PR TITLE
Bug 1232111 - Update the README file with a new skeleton and add a contributing doc.

### PR DESCRIPTION
**Pull request by Standard8**

### CODE REVIEW

File: docs/Contributing.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+------
+
+* If you didn't get review granted, or have comments to address, then edit the
+files. If there's significant changes, it helps to address the comments in a
+separate commit, to make the follow-up review easier.
+* Once you have been granted review, please add `r=<name>` onto the end of the
+commit message and rebase it against latest master so that it is ready to land.
+* If there's no comments that need addressing, once you've added the annotation
+then you can add checkin-needed onto the bug and someone will land it for you if
+it hasn't been already.
```
Comment (Mardak) [inline docs/Contributing.md:58]: I'm not sure if people will get grumpy with the `checkin-needed` keyword for non m-c patches, but I don't have any better ideas.

File: docs/Contributing.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+videos to the bug and request ui-review on those from :sevaan and :pau
+
+Review
+------
+
+* If you didn't get review granted, or have comments to address, then edit the
+files. If there's significant changes, it helps to address the comments in a
+separate commit, to make the follow-up review easier.
+* Once you have been granted review, please add `r=<name>` onto the end of the
+commit message and rebase it against latest master so that it is ready to land.
```
Comment (Mardak) [inline docs/Contributing.md]: nit: `s around `master`

File: docs/Contributing.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+which describes why it is being changed, for example:
+  * `Bug 123456 – Loop's RoomList view requires user profile data passed in when
+it doesn't need to.`
+* If you have multiple commits you should squash them down into one before
+creating a pull request, unless there's a reason for separating them, for example:
+  * With commits as one patch the patch would be a significant size and make it
+hard for review.
+  * When it makes sense to separate commits logically, e.g. “Implement a feature
+change”, “Remove code associated with previous version of feature”
+* Go to your repository in github, and create a pull request to the `master`
```
Comment (Mardak) [inline docs/Contributing.md:35]: Might be overly explicit, but one needs to push the branch to the github loop fork to see the pull request option.

File: docs/Contributing.md (hunk 1 above)
Comment (Standard8) [inline docs/Contributing.md:58]: I'll let the sheriffs know as they are the ones that have been handling it. They'll just start ignoring checkin-needed for us. Hopefully once we get autolander set up, that'll mean we won't have any tracking issues.

**Pull request merged**